<html><h3>:x: ci:test - 10 out of 13 jobs passed in 1 hour 9 minutes 19 seconds 430 ms</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/com-liferay-commerce/tree/7.1.x">7.1.x</a><br/>Branch GIT ID: <a href="https://github.com/liferay/com-liferay-commerce/commit/500cb59d9de82d9850311d20c6d7d8b4fbd90738">500cb59d9de82d9850311d20c6d7d8b4fbd90738</a></p>10 out of 13 jobs PASSED<div><h4>3 Failed Jobs:</h4><ul><li><a href="https://test-1-22.liferay.com/job/test-subrepository-acceptance-pullrequest(7.1.x)/252/"><strike><strong>test-subrepository-acceptance-pullrequest(7.1.x)</strong></strike></a></li><li><a href="https://test-1-19.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/599/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/2</strong></strike></a></li><li><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/423/"><strike><strong>test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/5</strong></strike></a></li></ul></div><details><summary><strong>10 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-19.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/600/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/functional-smoke-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/422/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-assemble-jdk8</a></li><li><a href="https://test-1-14.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/543/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/421/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/424/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-3.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/663/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-3.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/662/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-5.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/691/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-integration-mysql57-jdk8</a></li><li><a href="https://test-1-9.liferay.com/job/test-subrepository-acceptance-pullrequest-dist(7.1.x)/20/">test-subrepository-acceptance-pullrequest-dist(7.1.x)/jdk8</a></li><li><a href="https://test-1-18.liferay.com/job/test-subrepository-acceptance-pullrequest-validation(7.1.x)/101/">test-subrepository-acceptance-pullrequest-validation(7.1.x)/jdk8</a></li></ul></details><h5>For more details click <a href="https://test-1-22.liferay.com/userContent/jobs/test-subrepository-acceptance-pullrequest(7.1.x)/builds/252/jenkins-report.html">here</a>.</h5><hr/><h4>Failures (upstream comparison is not available):</h4><ol><li><div><h5><a href="https://test-1-19.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/599/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/2</a></h5><div><h6>Job Results:</h6><p>0 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-19.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/599//consoleText">AXIS_VARIABLE=0,label_exp=!master #599</a><pre><code>   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20190917165854173
[replaceregexp] The following file is missing: '/opt/dev/projects/github/liferay-jenkins-ee/testray/dependencies/test.generated.properties'
     [echo] Warning: Unable to locate '/opt/dev/projects/github/liferay-jenkins-ee/testray/dependencies/test.generated.properties'.
     [echo] Build testray result: /opt/dev/projects/github/liferay-jenkins-ee/testray/suite/TESTS-test-1-22_test-subrepository-acceptance-pullrequest(7.1.x)_252_subrepository-functional-tomcat90-mysql57-jdk8_2_0.xml
     [echo] send.to.testray.logs.start.timestamp: 09-17-2019 09:58:54:247 PDT
     [echo] Sending Testray logs to Testray production server.
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20190917165854252
[stopwatch] [send.to.testray.logs: 0.196 sec]
     [echo] send.to.testray.results.start.timestamp: 09-17-2019 09:58:54:444 PDT
     [echo] Sending Testray results to Testray production server.
   [delete] Deleting: /opt/dev/projects/github/liferay-jenkins-ee/commands/20190917165854447
[stopwatch] [send.to.testray.results: 0.150 sec]
[stopwatch] [record.testray.results.TESTS-test-1-22_test-subrepository-acceptance-pullrequest(7.1.x)_252_subrepository-functional-tomcat90-mysql57-jdk8_2_0.xml: 1.360 sec]

BUILD FAILED
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:11049: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-test-subrepository-acceptance-pullrequest-batch.xml:120: The following error occurred while executing this line:
/opt/dev/projects/github/liferay-jenkins-ee/commands/build-common.xml:8394: Sourced file: inline evaluation of: `` 								 									import com.liferay.jenkins.results.parser.GitHubDevSyncUtil;  . . . '' : Method Invocation gitWorkingDirectory.checkoutLocalGitBranch : at Line: 18 : in file: inline evaluation of: `` 								 									import com.liferay.jenkins.results.parser.GitHubDevSyncUtil;  . . . '' : gitWorkingDirectory .checkoutLocalGitBranch ( tempLocalGitBranch ) 

Called from method: checkoutUpstreamBranch : at Line: 110 : in file: inline evaluation of: `` 								 									import com.liferay.jenkins.results.parser.GitHubDevSyncUtil;  . . . '' : checkoutUpstreamBranch ( ) 
Target exception: java.lang.RuntimeException: Unable to checkout temp-1568739531742
fatal: unable to read tree 449d3e3088f6b80b410e6e4a6228adb235c517c0</code></pre></div></li></ol></div></li><li><div><h5><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/423/">test-subrepository-acceptance-pullrequest-batch(7.1.x)/subrepository-functional-tomcat90-mysql57-jdk8/5</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/423//consoleText">AXIS_VARIABLE=0,label_exp=!master #423</a><ol><li><div><a href="https://test-1-4.liferay.com/job/test-subrepository-acceptance-pullrequest-batch(7.1.x)/AXIS_VARIABLE=0,label_exp=!master/423//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_CommerceFrontStore_ViewStoreFrontSimpleProduct_">LocalFile.CommerceFrontStore#ViewStoreFrontSimpleProduct</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-22/1568736969995/test-subrepository-acceptance-pullrequest(7.1.x)/252/subrepository-functional-tomcat90-mysql57-jdk8/5/0/LocalFile.CommerceFrontStore_ViewStoreFrontSimpleProduct/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-22/1568736969995/test-subrepository-acceptance-pullrequest(7.1.x)/252/subrepository-functional-tomcat90-mysql57-jdk8/5/0/LocalFile.CommerceFrontStore_ViewStoreFrontSimpleProduct/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-22/1568736969995/test-subrepository-acceptance-pullrequest(7.1.x)/252/subrepository-functional-tomcat90-mysql57-jdk8/5/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li></ol></details></html>

--- a/commerce-product-test/src/testFunctional/macros/CommerceNavigator.macro
+++ b/commerce-product-test/src/testFunctional/macros/CommerceNavigator.macro
@@ -15,13 +15,10 @@ definition {
 	}
 
 	macro gotoFrontStoreProductPage {
-		Click(locator1 = "CommerceEntry#FRONT_STORE_PRODUCT_LINK");
+		var productName = StringUtil.replace("${productName}", " ", "-");
+		var productName = StringUtil.lowerCase("${productName}");
 
-		Pause(locator1 = "30000");
-
-		var siteName = TestCase.getSiteName(siteName = "${siteName}");
-
-		SelectWindow(locator1 = "title=${productName} - Product - ${siteName}");
+		Navigator.openSpecificURL(url = "localhost:8080/web/guest/p/${productName}");
 	}
 
 	macro gotoOptionValues {

--- a/commerce-product-test/src/testFunctional/paths/CommerceNavigation.path
+++ b/commerce-product-test/src/testFunctional/paths/CommerceNavigation.path
@@ -49,6 +49,11 @@
 	<td></td>
 </tr>
 <tr>
+	<td>SELECTABLE_CHECKBOX</td>
+	<td>//tr[@data-selectable="true"]//..//input[@type='checkbox']</td>
+	<td></td>
+</tr>
+<tr>
 	<td>ORGANIZATIONS_TABLE_CHECKBOX</td>
 	<td>//tr[@data-name="${key_orgName}"]//input[@type="checkbox"]</td>
 	<td></td>
@@ -56,6 +61,11 @@
 <tr>
 	<td>APPLY_BUTTON</td>
 	<td>//span[normalize-space()='${key_text}']</td>
+	<td></td>
+</tr>
+<tr>
+	<td>CHOOSE_SITE_BUTTON</td>
+	<td>//tr/td[contains(.,'${key_siteName}')]//..//button[contains(.,'Choose')]</td>
 	<td></td>
 </tr>
 </tbody>

--- a/commerce-product-test/src/testFunctional/tests/CommerceFrontStore.testcase
+++ b/commerce-product-test/src/testFunctional/tests/CommerceFrontStore.testcase
@@ -28,8 +28,8 @@ definition {
 
 		ProductMenu.gotoPortlet(
 			category = "Commerce",
-			panel = "Site Administration",
-			portlet = "Catalog");
+			panel = "Control Panel",
+			portlet = "Products");
 
 		echo("Creating new Option complete with Values...");
 

